remodel/connection.py

```diff
@@ -1,5 +1,5 @@
 
-import rethinkdb as r
+import rethinkdb
```

Comment:
But why go through all the hassle of separately importing the version when you could do this:

```
>>> import rethinkdb as r
>>> r.__version__
'2.3.0.post6'
```